OneSignalSDK/onesignal/src/main/java/com/onesignal/OSInAppMessageController.java

```diff
@@ -344,6 +347,45 @@ void onMessageActionOccurredOnPreview(@NonNull final OSInAppMessage message, @No
         logInAppMessagePreviewActions(action);
     }
 
+    /*
+    ** IAM Lifecycle methods
+    ** The following methods call the public inAppMessageLifecycleHandler callbacks
+    */
+
```

Comment:
Can we keep the same style to other Javadoc comments? 

/*
 *
 */
 method()